.github/workflows/build-release.yml

```diff
@@ -3,60 +3,49 @@ name: Publish release to PyPI
 on:
   release:
     types: [published]
-
-env:
-  INSTALL_EDM_VERSION: 3.3.1
+  workflow_dispatch:
```

Comment:
Do we want to keep this?

(I think "yes", but I'm genuinely asking)

Reply:
Yes, I think we want to keep the workflow dispatch for release building in case something goes wrong with the automatic build.